packages/http-client-csharp/generator/TestProjects/Local/Sample-TypeSpec/src/Generated/SampleTypeSpecClient.cs

```diff
@@ -1000,25 +1000,25 @@ public virtual async Task<ClientResult> GetUnknownValueAsync(string accept, Requ
         /// <param name="cancellationToken"> The cancellation token that can be used to cancel the operation. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="accept"/> is null. </exception>
         /// <exception cref="ClientResultException"> Service returned a non-success status code. </exception>
-        public virtual ClientResult<string> GetUnknownValue(string accept, CancellationToken cancellationToken = default)
+        public virtual ClientResult<GetUnknownValueResponse6> GetUnknownValue(string accept, CancellationToken cancellationToken = default)
```

Comment:
Is this expected?

Reply:
Yes this is expected.
Previously we did the conversion by ourselves and we only did it for model properties.
This is a literal in response, which I think we should also convert it to an enum.
I think in real services there should be no such case like this.